pkg/controller/cainjector/setup.go

```diff
@@ -93,19 +93,19 @@ func registerAllInjectors(ctx context.Context, groupName string, mgr ctrl.Manage
 	g, gctx := errgroup.WithContext(ctx)
 
 	g.Go(func() (err error) {
-		if err = ca.Start(gctx.Done()); err != nil {
```

Comment:
Note to self: this (and below) is done because in controller-runtime `v0.7.0` the `Start` function was changed to accept a context with cancel instead of stop channel, see [release notes](https://github.com/kubernetes-sigs/controller-runtime/releases/tag/v0.7.0).
This is fine as `gctx` is a context with cancel.